src/common/low_precision_transformations/src/fake_quantize_dequantization.cpp

```diff
@@ -25,16 +25,14 @@ FakeQuantizeDequantization::FakeQuantizeDequantization(
     const std::shared_ptr<ov::opset1::Constant>& subtractConstant,
     const std::shared_ptr<opset1::Multiply>& multiply,
     const std::shared_ptr<ov::opset1::Constant>& multiplyConstant) :
+    channelDimIndex(1ul),
```

Comment:
Do we need to set it twice? (In class definition and here)

Reply:
I suppose we don't. The duplication is removed, thanks